binary_transparency/firmware/api/witness.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2020 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package api
+
+const (
+	//WitnessGetCheckpoint is the path of the URL to get witness checkpoint.
```

Comment:
nit: Comments in Go should have a space between the `//` and the text. This applies throughout the PR but I'll only comment on it this once to save us both a lot of writing and reading!

Reply:
Yes, Done everywhere, thanks for the spot!! :)